src/style-spec/reference/v8.json

```diff
@@ -5013,6 +5013,27 @@
       },
       "property-type": "data-constant"
     },
+    "raster-scaling": {
+      "type": "enum",
+      "doc": "The raster scaling to use for overscaling",
```

Comment:
Consider: "The raster scaling interpolation technique to use for overscaling". Is _interpolation technique_ the right word here?